### PR TITLE
Merge hawthorn

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

File: notifier/pull.py [hunk 1]
```diff
@@ -87,30 +87,42 @@ def _build_digest(user_content, user_info):
         )
     )
 
+def _should_skip_org(course_id):
+    # TODO: Need to implement an end point in LMS to get this list.
+    orgs_to_skip = ["WhartonOnlineProfessionalEd"]
```
Comment (stvstnfrd) [inline notifier/pull.py:92]: This is hilariously bad...

File: Makefile [hunk 2]
```diff
@@ -9,10 +9,10 @@ coverage:
 	coverage run manage.py test notifier
 	coverage report
 
-quality: pep8 pylint
+quality: pycodestyle pylint
```
Comment (stvstnfrd) [inline Makefile:12]: PEP8 has been renamed..

**Comment by stvstnfrd** [comment]
@kluo BTW: it looks like this ought to be backwards compatible, so we can probably release it out-of-band and in advance. Thoughts?

**Review by kluo — APPROVED** [review]
Yea looks fine to release in advance

**Pull request merged**